dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1295,9 +1278,9 @@ def eye(
 
     Limitations
     -----------
-    Parameter `order` is supported only with values ``"C"`` and ``"F"``.
+    Parameter `order` is supported only with values ``"C"``, ``"F"`` and ``None``.
```

Comment:
Actually, at least for `dpnp.eye`, numpy has the same range of order values: "C", "F", None.
In that case I don't think we need to state that as a limitation in dpnp then: the range is exactly the same.

In contrast to the comment I left about `dpnp.empty_like`, the order  in `numpy.empty_like` accepts also "A" and "K", but dpnp doesn't. And so in that case we need to state such limitation.